### ISSUE TITLE
Stable baselines version

### ISSUE THREAD

**Issue by bbalaji-ucsd** [PCCproject/PCC-RL#10]
Which version of stable-baselines does the repository support? I could not get it to run with the latest version. 
```
Traceback (most recent call last):
  File "stable_solve.py", line 21, in <module>
    from stable_baselines import PPO1
ImportError: cannot import name 'PPO1'
```

**Comment by hexi519** [comment]
I have a question alike: what's the required configuration of the environment ... Maybe a requirement.txt is better.

**Comment by r02b** [comment]
Yes, a requirement list is probably a good idea. With that being said, the OP's error seems to be stemming from a stable_baselines problem:
https://stable-baselines.readthedocs.io/en/master/modules/ppo1.html

The rest (other than legacy TF1.14) should be fine.

**Comment by jessewjx** [comment]
Hello,

I seem to have encountered the same issue here, I am wondering how you resolved this issue?
Thank you very much!

I am using python3.7, tensorflow==1.15, stable-baselines==2.10.0, gym==0.18.0
/usr/local/lib/python3.7/site-packages/sklearn/utils/validation.py:37: DeprecationWarning: distutils Version classes are deprecated. Use packaging.version instead.
  LARGE_SPARSE_SUPPORTED = LooseVersion(scipy_version) >= '0.14.0'
WARNING: Logging before flag parsing goes to stderr.
W0409 20:02:53.006458 4432291328 lazy_loader.py:50] 
The TensorFlow contrib module will not be included in TensorFlow 2.0.
For more information, please see:
  * https://github.com/tensorflow/community/blob/master/rfcs/20180907-contrib-sunset.md
  * https://github.com/tensorflow/addons
  * https://github.com/tensorflow/io (for I/O related ops)
If you depend on functionality not listed there, please file an issue.

Traceback (most recent call last):
  File "stable_solve.py", line 21, in <module>
    from stable_baselines import PPO1
ImportError: cannot import name 'PPO1' from 'stable_baselines' (/usr/local/lib/python3.7/site-packages/stable_baselines/__init__.py)

Best,
Juexiao

**Comment by anilsh** [comment]
Installing mpi4py worked for me. `pip insall mpi4py` 
If the above command runs in error `mpi.h` not found then run `sudo apt install libopenmpi-dev`
